### PR TITLE
Querrier support for req-res delay

### PR DESCRIPTION
**Pull request by aquibbaig**
Signed-off-by: Aquib Baig <aquibbaig97@gmail.com>

### CODE REVIEW

**Comment by aquibbaig** [comment]
@muskankhedia Can you test this latest piece here and let me know? I've been a bit occupied lately

**Comment by muskankhedia** [comment]
@aquibbaig Yaa sure.

File: src/main.go [hunk 1]
```diff
@@ -284,7 +284,9 @@ func main() {
 				method := inst.Method
 				// Gets the Querrier for request-response delay
 				// TODO: Send the method along with URL
+				fmt.Println("Reached1")
 				ql := getQuerier(ws, "req-res-delay", url, method, "_delay")
+				fmt.Println("Reached")
```
Comment (aquibbaig) [inline src/main.go]: @muskankhedia Kindly remove *fmt* statement before pushing the code

File: src/main.go [hunk 2]
```diff
@@ -182,7 +182,7 @@ func main() {
 
 			sig := inStream[0] // Signal
 			log.Printf("type: %d\n message: %s \n", messageType, sig)
-
+			fmt.Println("asdfa:::", sig)
```
Comment (aquibbaig) [inline src/main.go]: Here too!

**Pull request merged**